cmake/FindOGG.cmake

```diff
@@ -0,0 +1,39 @@
+find_package(PkgConfig)
+pkg_check_modules(_OGG QUIET ogg)
+
+find_path(OGG_INCLUDE_DIR
+    NAMES "ogg/ogg.h"
+    PATHS ${_OGG_INCLUDE_DIRS})
+
+find_library(OGG_LIBRARY
+    NAMES ogg libogg
```

Comment:
I'm pretty sure `libogg.a` will be found here under Linux (if only static library is installed).

Reply:
I've made FLAC and FLAC++ depend on BUILD_SHARED_LIBS so now there is no explicit division on static and shared libs. As a result FindOGG.cmake now only searches for any OGG lib and adds UNKNOWN target. Seems that should fit all the setups.